BUILDING.md

```diff
@@ -56,6 +56,15 @@ your distribution. Here's a list.
 - [Graphviz](http://www.graphviz.org/) - required to build the API documentation and include fancy class diagrams
 - [libgps](https://gpsd.gitlab.io/gpsd/index.html) - if you want to build Stellarium with GPS support (Linux/macOS only)
 
+### Optionally bundled dependencies
+
+If these are not found on the system, they are downloaded automatically.
+
+- [QCustomPlot](https://www.qcustomplot.com)
```

Comment:
Please remove this line, because QCustomPlot is not used anymore

Reply:
Oh, right.

Removed QCompress too, since switched to the internal QGui's one.